### PR TITLE
harfbuzz: add --HEAD

### PR DESCRIPTION
**Pull request by moyogo**

### CODE REVIEW

File: Library/Formula/harfbuzz.rb [hunk 1]
```diff
@@ -35,6 +44,10 @@ def install
       --with-coretext=yes
     ]
 
+    if build.head?
+      system "./autogen.sh"
+    end
```
Comment (bfontaine) [inline Library/Formula/harfbuzz.rb]: You can inline that: `system "./autogen.sh" if build.head?`.

Comment (moyogo) [inline Library/Formula/harfbuzz.rb]: Updated and squashed. Thanks.

File: Library/Formula/harfbuzz.rb [hunk 2]
```diff
@@ -35,6 +44,8 @@ def install
       --with-coretext=yes
     ]
 
+    system "./autogen.sh" if build.head?
```
Comment (DomT4) [inline Library/Formula/harfbuzz.rb]: Can you put this above `system "./configure", *args` line instead of here. Thanks.

Comment (moyogo) [inline Library/Formula/harfbuzz.rb]: Done. Thanks.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/5b03d91ce96c5ea9bf44330a7491ee014b39eafc. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Comment by moyogo** [comment]
Thank you!

**Pull request closed without merging**